elado/yacht_kikoto.index.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: elado
+title: Új tulajdonosra vár a színvonalas, népszerű, balatoni yacht kikötő!
+---
+
+#![Kívülről](http://i.imgur.com/4232j52.jpg) Új tulajdonosra vár a színvonalas, népszerű, balatoni yacht kikötő!
```

Comment:
Persze, elég a rövid cím, átírom...

A "Kívülről"-ről azt hittem, hogy azért kell, mert valahonnan, itt pl. az
Imgur-ból importálom a képet.
Eddig is minden kép előtt szerepelt.
Egyszerűen csak hagyjam ki?

Jól vagytok? Félix élvezte az esküvőt? :)